### PR TITLE
translate

### PR DESCRIPTION
**Pull request by chloeyutianyi**
Added translate.html page for future translation use.
Changed aboutus.html: add translate button, after click on the button about our team label would be changed into Chinese. Language code is hardcoded so the label can only be changed into Chinese. I figured out that setting each line into a label and set the text after clicking on the translate button is inefficient, so I only change the title into a label as an example. May need to explore other API instead.

### CODE REVIEW

File: src/main/webapp/WEB-INF/translate.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
```
Comment (ginellegaisano) [inline src/main/webapp/WEB-INF/translate.html:1]: I'm a little unclear as to why this is in the WEB-INF directory? Also, only Chinese is currently supported, right?

File: src/main/webapp/js/translate-aboutus-loader.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+/**
```
Comment (ginellegaisano) [inline src/main/webapp/js/translate-aboutus-loader.js:1]: It seems like this code is very similar between the two loader.js and and the inline javascript in translate.html. Perhaps you could make the requestTranslation() function more generic and use it throughout.

File: src/main/webapp/WEB-INF/translate.html (hunk 1 above)
Comment (ginellegaisano) [reply src/main/webapp/WEB-INF/translate.html:1]: Also, javascript and css should be in a separate file.

File: src/main/webapp/css/translate-page.css [hunk 3]
```diff
@@ -0,0 +1,11 @@
+@charset "UTF-8";
+#text {
+	width: 500px;
```
Comment (ginellegaisano) [inline src/main/webapp/css/translate-page.css]: nit: fix indentation

File: src/main/webapp/js/translate-loader.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
+/**
+ * http://usejsdoc.org/
+ */
+
+      function requestTranslation() {
```
Comment (ginellegaisano) [inline src/main/webapp/js/translate-loader.js]: nit: fix indentation

File: src/main/webapp/js/translate-aboutus-loader.js (hunk 2 above)
Comment (chloeyutianyi) [reply src/main/webapp/js/translate-aboutus-loader.js:1]: Yes the two files are similar. Translate-loader is for a separate page which I think we may not need it in the future so I separate the two files.

**Review by ginellegaisano — APPROVED** [review]

**Pull request merged**